qiskit/visualization/pulse_v2/device_info.py

```diff
@@ -106,40 +106,65 @@ def create_from_backend(cls, backend: Backend):
         Returns:
             OpenPulseBackendInfo: New configured instance.
         """
-        configuration = backend.configuration()
-        defaults = backend.defaults()
-
-        # load name
-        name = backend.name()
-
-        # load cycle time
-        dt = configuration.dt
-
-        # load frequencies
         chan_freqs = {}
-
-        chan_freqs.update(
-            {pulse.DriveChannel(qind): freq for qind, freq in enumerate(defaults.qubit_freq_est)}
-        )
-        chan_freqs.update(
-            {pulse.MeasureChannel(qind): freq for qind, freq in enumerate(defaults.meas_freq_est)}
-        )
-        for qind, u_lo_mappers in enumerate(configuration.u_channel_lo):
-            temp_val = 0.0 + 0.0j
-            for u_lo_mapper in u_lo_mappers:
-                temp_val += defaults.qubit_freq_est[u_lo_mapper.q] * u_lo_mapper.scale
-            chan_freqs[pulse.ControlChannel(qind)] = temp_val.real
-
-        # load qubit channel mapping
         qubit_channel_map = defaultdict(list)
-        for qind in range(configuration.n_qubits):
-            qubit_channel_map[qind].append(configuration.drive(qubit=qind))
-            qubit_channel_map[qind].append(configuration.measure(qubit=qind))
-            for tind in range(configuration.n_qubits):
+
+        if hasattr(backend, "configuration") and hasattr(backend, "defaults"):
+            configuration = backend.configuration()
+            defaults = backend.defaults()
+
+            name = configuration.backend_name
+            dt = configuration.dt
+
+            # load frequencies
+            chan_freqs.update(
+                {
+                    pulse.DriveChannel(qind): freq
+                    for qind, freq in enumerate(defaults.qubit_freq_est)
+                }
+            )
+            chan_freqs.update(
+                {
+                    pulse.MeasureChannel(qind): freq
+                    for qind, freq in enumerate(defaults.meas_freq_est)
+                }
+            )
+            for qind, u_lo_mappers in enumerate(configuration.u_channel_lo):
+                temp_val = 0.0 + 0.0j
+                for u_lo_mapper in u_lo_mappers:
+                    temp_val += defaults.qubit_freq_est[u_lo_mapper.q] * u_lo_mapper.scale
+                chan_freqs[pulse.ControlChannel(qind)] = temp_val.real
+
+            # load qubit channel mapping
+            for qind in range(configuration.n_qubits):
+                qubit_channel_map[qind].append(configuration.drive(qubit=qind))
+                qubit_channel_map[qind].append(configuration.measure(qubit=qind))
+                for tind in range(configuration.n_qubits):
+                    try:
+                        qubit_channel_map[qind].extend(configuration.control(qubits=(qind, tind)))
+                    except BackendConfigurationError:
+                        pass
+        elif isinstance(backend, BackendV2):
```

Comment:
Lint is complaining about this being an elif instead of an else. I think it is reasonable to assume that the default is `BackendV2` and not do the instance check:

```suggestion
        else:
```

Reply:
The other thing we sometimes do is to be more defensive and do the check, in order to raise an explicit `RuntimeError("backend object not yet supported")` for a clearer message - we don't have a `BackendV3`, but it's not inconceivable that we will expand in the future and have this same bug again.